packaging/macos/launcher-mono.m

```diff
@@ -359,6 +343,35 @@ - (void)taskExited:(NSNotification *)note
 int main(int argc, char **argv)
 {
 	NSAutoreleasePool *pool = [[NSAutoreleasePool alloc] init];
+
+	if (argc > 1)
+	{
+		struct rlimit limit;
+		if (getrlimit (RLIMIT_NOFILE, &limit) == 0 && limit.rlim_cur < 1024)
+		{
+			limit.rlim_cur = MIN(limit.rlim_max, 1024);
+			setrlimit(RLIMIT_NOFILE, &limit);
+		}
+
+		void *libmono = dlopen([[SYSTEM_MONO_PATH stringByAppendingPathComponent: @"/lib/libmonosgen-2.0.dylib"] UTF8String], RTLD_LAZY);
```

Comment:
libmonosgen-2.0.dylib or libmono-2.0.dylib?

Reply:
libmonosgen still appears to be correct, based on the latest version of https://github.com/mono/monodevelop/blob/main/main/build/MacOSX/monostub.mm.